src/publisher/message-queues.ts

```diff
@@ -309,7 +307,10 @@ export class OrderedQueue extends MessageQueue {
     const timeWaiting = Date.now() - this.currentBatch.created;
     const delay = Math.max(0, maxMilliseconds - timeWaiting);
 
-    this.pending = setTimeout(() => this.publish(), delay);
+    this.pending = setTimeout(() => {
+      // Ignore errors.
+      this.publish().catch(() => {});
```

Comment:
What's the motivation for ignoring errors?

Reply:
Discussed in chat, but copied here for posterity: I was mostly just emulating the behaviour of the old code, since it doesn't pass an error callback, but this does still change semantics a bit. (Code exceptions won't bubble up now.) I'm going to see if I can make this a bit more closely scoped first.